### PR TITLE
FILEMAN-130: User needs edit right in order to do a multiple file download

### PR DESCRIPTION
**Pull request by msmeria**

### CODE REVIEW

File: ui/src/main/resources/FileManagerCode/DriveSheet.xml [hunk 1]
```diff
@@ -133,7 +133,11 @@
   #set ($jobId = $services.drive.pack($paths, $packReference))
   #if ($jobId)
     #set ($discard = $downloadDoc.set('jobId', $jobId))
-    #set ($discard = $downloadDoc.saveAsAuthor())
+    #if ($xwiki.checkAccess($downloadDoc, "edit"))
```
Comment (mflorea) [inline ui/src/main/resources/FileManagerCode/DriveSheet.xml]: See http://maven.xwiki.org/site/docs/xwiki-javadoc-4.1.x/com/xpn/xwiki/api/XWiki.html#checkAccess(java.lang.String,%20java.lang.String) .

> If the document requires rights and the user is not authenticated he will be redirected to the login page.

Better use http://maven.xwiki.org/site/docs/xwiki-javadoc-4.1.x/com/xpn/xwiki/api/Document.html#hasAccessLevel(java.lang.String) .

File: ui/src/main/resources/FileManagerCode/DriveSheet.xml [hunk 2]
```diff
@@ -133,7 +133,7 @@
   #set ($jobId = $services.drive.pack($paths, $packReference))
   #if ($jobId)
     #set ($discard = $downloadDoc.set('jobId', $jobId))
-    #if ($xwiki.checkAccess($downloadDoc, "edit"))
+    #if ($xwiki.hasAccessLevel("edit", $xcontext.getUser() , $downloadDoc))
```
Comment (mflorea) [inline ui/src/main/resources/FileManagerCode/DriveSheet.xml]: Read again my comment :)

> Better use http://maven.xwiki.org/site/docs/xwiki-javadoc-4.1.x/com/xpn/xwiki/api/Document.html#hasAccessLevel(java.lang.String) .

Don't you think the method I indicated is simpler?

Comment (vmassol) [inline ui/src/main/resources/FileManagerCode/DriveSheet.xml]: Isn't it better to use $services.security.authorization.hasAccess(...) ?

Comment (msmeria) [inline ui/src/main/resources/FileManagerCode/DriveSheet.xml]: $services.security.authorization.hasAccess(...) has been introduced in 6.1RC1 according to http://extensions.xwiki.org/xwiki/bin/view/Extension/Security+Module#HScriptingAPI28since6.1RC129

File Manager needs to be compatible with XWiki versions starting with 5.4.4 -> https://github.com/xwiki-contrib/application-filemanager/blob/master/pom.xml#L30

I've sent this new commit https://github.com/xwiki-contrib/application-filemanager/pull/31/commits/ffb8a6945965dd3413c321193c52d1a9b36fbe6c which will hopefully please everyone :)

**Pull request merged**